source/common/upstream/load_balancer_impl.h

```diff
@@ -469,7 +487,9 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
   }
   HostConstSharedPtr unweightedHostPick(const HostVector& hosts_to_use,
                                         const HostsSource& source) override;
+
   const uint32_t choice_count_;
+  bool alternativeWeights_;
```

Comment:
Also explain what this means exactly in a comment. It's non-obvious from the name.